radix-engine/src/blueprints/resource/non_fungible/non_fungible_proof.rs

```diff
@@ -145,8 +145,7 @@ impl NonFungibleProofBlueprint {
     where
         Y: ClientApi<RuntimeError>,
     {
-        let address =
-            ResourceAddress::new_or_panic(api.actor_get_object_info()?.get_outer_object().into());
+        let address = ResourceAddress::new_or_panic(api.actor_get_outer_object()?.into());
```

Comment:
Minor: I noticed that we sometimes `unwrap` (in native-sdk) and other times propagate the error.